eclipse/plugin/src/nl/tudelft/watchdog/eclipse/ui/wizards/ProjectRegistrationInputPanel.java

```diff
@@ -43,35 +78,108 @@
 		new Composite(inputContainer, SWT.NONE);
 
-		this.codeStyleUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(CODE_STYLE_USAGE_LABEL_TEXT, inputContainer);
-		this.bugFindingUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(BUG_FINDING_USAGE_LABEL_TEXT, inputContainer);
-		this.automationUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(OTHER_AUTOMATION_USAGE_LABEL_TEXT, inputContainer);
+		this.codeStyleUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(CODE_STYLE_USAGE_LABEL_TEXT,
+				inputContainer);
+		this.bugFindingUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(BUG_FINDING_USAGE_LABEL_TEXT,
+				inputContainer);
+		this.automationUsage = RegistrationStep.createYesNoDontKnowQuestionWithLabel(OTHER_AUTOMATION_USAGE_LABEL_TEXT,
+				inputContainer);
+
+		this.toolsUsed = RegistrationStep.createLinkedLabelTextField(TOOL_USAGE_LABEL_TEXT,
+				TOOL_USAGE_TEXTFIELD_TOOLTIP, inputContainer);
+	}
 
-		this.toolsUsed = RegistrationStep.createLinkedLabelTextField(TOOL_USAGE_LABEL_TEXT, TOOL_USAGE_TEXTFIELD_TOOLTIP, inputContainer);
+	private void createSlider(Composite composite) {
+		UIUtils.createLabel("", composite);
+		UIUtils.createLabel(
+				"Estimate how you divide your time into the two activities testing and production. Just have a wild guess!",
+				composite);
+
+		this.productionPercentageStart = ThreadLocalRandom.current().nextInt(0, 100 + 1);
+		this.sliderRow = UIUtils.createFullGridedComposite(composite, 3);
+
+		Label testingLabel = UIUtils.createLabel("100% Testing  ", sliderRow);
+		testingLabel.setToolTipText(
+				"To the testing activity, everything you do with Junit tests counts. Examples: writing, modifying, debugging, and executing Junit tests");
+		this.percentageProductionSlider = new Scale(sliderRow, SWT.HORIZONTAL);
+		this.percentageProductionSlider.setLayoutData(UIUtils.createFullGridUsageData());
+		this.percentageProductionSlider.setSelection(this.productionPercentageStart);
+		this.percentageProductionSlider.setIncrement(5);
+		this.percentageProductionSlider.setPageIncrement(5);
+		this.percentageProductionSlider.setMaximum(100);
+		this.percentageProductionSlider.setMinimum(0);
+		Label productionLabel = UIUtils.createLabel("  100% Production", sliderRow);
+		productionLabel.setToolTipText(
+				"To the production activity, every activity that has to do with regular, non-test production code counts.");
+		UIUtils.createLabel("", sliderRow);
+		final Label sliderValueText = UIUtils.createItalicLabel("", sliderRow);
+		sliderValueText.setLayoutData(UIUtils.createFullGridUsageData());
+		sliderValueText.setAlignment(SWT.CENTER);
+
+		SelectionListener sliderMovedListener = new SelectionListener() {
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+				Color backgroundColor = sliderComposite.getParent().getBackground();
+				sliderComposite.setBackground(backgroundColor);
+				sliderRow.setBackground(backgroundColor);
```

Comment:
I haven't ran the code, but what are these colors used for? Maybe add a clarifying comment with the reasoning.